### PR TITLE
94565-fix-skip-duplicate-services-bug

### PR DESCRIPTION
**Pull request by feliara8**
- [x] @drewburlingame

### CODE REVIEW

**Comment by feliara8** [comment]
@drew can you help me where i have to update the runkey with the messageId?

File: Druq.Framework/Model/RunKey.cs [hunk 1]
```diff
@@ -8,5 +8,6 @@ public class RunKey
         public DateTime? LastRun { get; set; }
         public Guid? CurrentHubId { get; set; }
         public Guid? IntraHubLock { get; set; }
+        public Guid MessageId { get; set; }
```
Comment (drewburlingame) [inline Druq.Framework/Model/RunKey.cs]: Guid?

**Comment by feliara8** [comment]
@drewburlingame changes pushed, ive tested enough and no red tests again popped, please comment if you found anything again

**Comment by feliara8** [comment]
Thanks @drewburlingame , i think its ready to test now, what do you think?

**Comment by drewburlingame** [comment]
yes, i think so

**Comment by drewburlingame** [comment]
:rocket:

**Pull request merged**